src/file/file.service.ts

```diff
@@ -172,11 +172,12 @@ export class FileService {
    * @returns {IFile[]}
   */
   public static async getFilesByFolder(folderID: string | null, ownerID: string | null, deleted = false): Promise<IFile[]> {
+    if (!ownerID) throw new ClientError('No file or owner id sent');
+
     let files: IFile[];
```

Comment:
Should replace all lines after the `ownerID` check with something like this:
```
const parent = folderID ? new ObjectID(folderID) : null 
return FilesRepository.find({ deleted, ownerID, parent }).exec()
```

Reply:
Don't just copy the example code without changing anything, there's a missing `;` at the end of the first line and missing new line before the `return` statement.

Reply:
The `if` block should have curly braces (`{}`), regardless of it having a single line of code or not, and there should be a new line after it.